README.md

```diff
@@ -10,7 +10,11 @@ var credentials = require('plex-api-credentials');
 
 var userAndPass = credentials({
     username: 'foo',
-    password: 'bar'
+    password: 'bar',
+    homeUser: {    // Required for Plex managed users
```

Comment:
Should we call this `managedUser`? That would seem more inline with what Plex calls it when these managed users in a Plex Home environment.

![image](https://cloud.githubusercontent.com/assets/1231635/16096169/0d203f12-3348-11e6-903e-554e3fe507cc.png)